config/routes.rb

```diff
@@ -7,7 +7,7 @@
     end
   end
   resources :questions
-  devise_for :users
+  devise_for :users, :controllers => {:registrations => "users/registrations"}
```

Comment:
is `users/registrations` a valid route? I dont see it in the list after running `rails routes`?

Reply:
I know. I didn't understand it completely too but based on my research we need to tell rails to use the overwritten methods instead of devise's defaults and it is a way of doing that. Without it our overwritten method won't be called.

Reply:
that is really strange..... since if we have a method associated with this route, we should see it in the rails routes command

Could you link the source you used? Maybe even adding a comment regarding this in the code will be helpful.